code/modules/economy/ATM.dm

```diff
@@ -68,7 +68,7 @@ log transactions
 		break
 
 /obj/machinery/atm/emag_act(var/remaining_charges, var/mob/user)
```

Comment:
Imho you should bump the amount of money spawned quite a bit.
100-500 credits wont bother anyone.